### ISSUE TITLE
Keyboard shortcut for new email

### ISSUE THREAD

**Issue by mikeperstin** [Thomas101/wmail#507]
It would be really useful and convenient to add a "Compose New Email" keyboard shortcut so that by using a combination of keys when Wmail is not in focus, it will come up and allow the user to start a new email (or open a separate window for it). For example, Mailbird and Outlook allow you to configure a combination of keys, that when used at anytime (e.g. Ctrl+Alt+N) will open a new email pop up and allow you to quickly compose and send a new email.

**Comment by Thomas101** [comment]
Yeah I agree! I've already got the code to trigger all that so will try to add in the next release
